src/data-access/repositories/test/mongo-part-repository.ts

```diff
@@ -12,11 +12,7 @@ import { EntityTestFactory } from '../../../model/entities/test/entity-test-fact
 
 const COLLECTION_NAME = 'parts'
 
-describe(MongoPartRepository.name, () => {
-  const testDatabase: MongoTestDatabase = new MongoTestDatabase()
-  beforeEach(async () => testDatabase.setupDatabase())
-  afterEach(async () => testDatabase.teardownDatabase())
-
+export function MongoPartRepositoryTests(testDatabase: MongoTestDatabase): void {
```

Comment:
The method name sounds like a collection of tests. To indicate that this is a function that runs the tests, it could be renamed to `testMongoPartRepository` or `runTestsForMongoPartRepository`.

Reply:
Prefixed `Run-` to the function names.

Reply:
We use camelCase for function names, so r in Run should be lowercased.